### PR TITLE
fix: Don't skip title, body from final DataFrame

### PR DESCRIPTION
**Pull request by rajusem**
- Saving title & body with the csv files which will be used to insert data to db via ap-server

### CODE REVIEW

**Comment by rajusem** [comment]
[test]

File: utils/storage_utils.py [hunk 1]
```diff
@@ -53,9 +55,11 @@ def write_output_csv(start_time, end_time, cve_model_type, ecosystem, df, s3_upl
         "closed_at",
         "creator_name",
         "creator_url",
+        "ecosystem",
+        "title",
```
Comment (rootAvish) [inline utils/storage_utils.py:59]: Are these columns "title" and "body" available in this data frame already?

Comment (rajusem) [reply utils/storage_utils.py:59]: Yes they are available

**Review by rootAvish — APPROVED** [review]

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**